test/binding.gyp

```diff
@@ -28,6 +28,11 @@
         'movable_callbacks.cc',
         'memory_management.cc',
         'name.cc',
+        'globalObject/delete_property.cc',
+        'globalObject/has_own_property.cc',
+        'globalObject/set_property.cc',
+        'globalObject/get_property.cc',
+        'globalObject/object.cc',
```

Comment:
The build for the test fails on windows. In windows I'm getting the error reported below:
![build-win](https://user-images.githubusercontent.com/387506/112338172-dd55e180-8cbe-11eb-8ee7-6e1afc072793.PNG)
The problem is that on windows the object files will be created on `build/Release/obj/binding` or `build/Release/obj/binding_noexcept` instead in unix system the object files will be created on the subfolders you set. 
One solution for me  was to chenge the name of the following files:
```
 'globalObject/delete_property.cc',
 'globalObject/has_own_property.cc',
 'globalObject/set_property.cc',
 'globalObject/get_property.cc',
  'globalObject/object.cc',
```
to something like this:

```
 'globalObject/global_object_delete_property.cc',
 'globalObject/global_object_has_own_property.cc',
 'globalObject/global_object_set_property.cc',
 'globalObject/global_object_get_property.cc',
  'globalObject/global_object.cc',
```
This avoid the name conflicts for the object files generated on windows system.

Reply:
Okay thank you! I am just wondering, i noticed that we run the tests per PR on MacOS and ubuntu , should we setup a similar CI for a window systems to catch issues like this?

Reply:
I agree with you and I just opened a new PR https://github.com/nodejs/node-addon-api/pull/948 that add the windows platform to CI.